vllm/compilation/collective_fusion.py

```diff
@@ -154,7 +156,7 @@ def __call__(self, graph: fx.Graph):
     # Max size of the input tensor per world size
     # to use flashinfer fused allreduce
     _FI_MAX_SIZES = {
-        2: MiB,  # 1MB
+        2: 64 * MiB,  # 64MB
         4: MiB,  # 1MB
```

Comment:
@ilmarkov Will you also adjust the thresholds for TP4/6/8? Thanks!

Reply:
yes, I meant are you planning to run that sweeping by yourself some time in the future?